src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/SolutionPickerItemViewModel.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class SolutionPickerItemViewModel
+    {
+        public SolutionPickerItemViewModel(ICommand command, string filePath)
+        {
+            OpenSolutionCommand = command;
+            FullPath = filePath;
```

Comment:
Is this necessary to store? What's it used for? Otherwise we may be fine just storing the directory name and the file name individually if that's all we're using.

Reply:
It is being used to aid in opening the sln file at that path when the button is clicked, this property is binded on the SolutionPicker xaml. My button displays the directory name and the sln file at said directory separately so I needed to store a property that encapsulates the two.